IdleServer.js

```diff
@@ -52,11 +46,11 @@ const SERVER_CONFIG = {
 const PLAYFAB_ENVIRONMENT_CONFIG = {
 	configs: {
 		"18A45": {
-			apiKey: "C5NQN8YPPCOF43HSJEWOJNZ9Z7FI9IMSIZDGU7H1CZEMJTNZT6", // playfab auto
+			apiKey: "ORX7Y8EKSK5DD1MOKUE86X5D64RQ774359ICEJJMN7U176GOBF", // playfab auto
```

Comment:
I think we do not need this either, at least until we have to hit outside the server sdk or cloud script helper objects